common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -0,0 +1,209 @@
+@special_maa_01 = "gfx/interface/icons/culture_innovations/innovation_special_maa_01.dds"
+@special_maa_02 = "gfx/interface/icons/culture_innovations/innovation_special_maa_02.dds"
+
+#culture_group_regional
+####################
+##Tribal##
+####################
+innovation_amphibious = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @misc_inventions
+	potential = {
+		OR = {
+			##Nagas
+			this = culture:naga
+			##Murlocs
+			this.culture_group = culture_group:murloc_group
+			##Giblin
+			this = culture:giblin
+		}
+	}
+	modifier = {
+		embarkation_cost_mult = -1
+		no_water_crossing_penalty = yes
+		movement_speed = -0.155
+		supply_duration = -0.25
+	}
+	custom = unlocks_naval_raiding
+	custom = unlocks_sailable_major_rivers
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_sea_faring = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @misc_inventions
+	potential = {
+		OR = {
+			##Human
+			this = culture:tirassian
+			##Vrykul
+			this.culture_group = culture_group:vrykul_group
+			##Pirates
+			this = culture:pirate_culture
+			this = culture:blackwater
+			##Trolls
+			this.culture_group = culture_group:troll_group
+			##Ogres
+			this.culture_group = culture_group:ogre_group
+		}
+	}
+	modifier = {
+		naval_movement_speed_mult = 0.25
+		embarkation_cost_mult = -0.75
+		garrison_size = -0.30
+	}
+	custom = unlocks_naval_raiding
+	custom = unlocks_sailable_major_rivers
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_lorewalking = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @misc_inventions
+	potential = {
+		 OR = {
+			##Pandaren
+			this.culture_group = culture_group:pandaren_group
+		  }
+	}
+	modifier = {
+		monthly_lifestyle_xp_gain_mult = 0.05
+	}
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_hive_breeding = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @misc_inventions
+	potential = {
+		 OR = {
+			##Aqir
+			this.culture_group = culture_group:aqir_group
+		}
+	}
+	modifier = {
+		levy_reinforcement_rate = 0.2
+	}
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+
+####################
+##Early Medieval##
+####################
+innovation_time_is_money = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @misc_inventions
+	potential = {
+		##Goblin
+		this = culture:goblin	
+	}
+	modifier = {
+		monthly_income_mult = 0.05
+		vassal_tax_contribution_mult = 0.1
+	}
+
+	flag = global_regional
+	flag = early_medieval_era_regular
+}
+innovation_elite_ranks = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @misc_inventions
+	potential = {
+		  OR = {
+			##Human
+			this.culture_group = culture_group:human_group
+		  NOT = {
+			this = culture:wastewander
+			this = culture:pirate_culture
+			
+			}
+	
```

Comment:
The condition is borken. Everyone can get Elite Ranks.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103773515-9c95e700-5044-11eb-8d99-36a08c6f1fac.png)

</details>